### DISCUSSION TITLE
Jetson AGX Xavier Development Kit Boot Issues on Kirkstone 5.10

### DISCUSSION THREAD

**Discussion by mwest90** [OE4T/meta-tegra#1089] [Q&A]
Hi,

We have a project based on this meta-tegra image with some custom layers on top.
Previously we had been running an older version of the Kirkstone branch with kernel 4.9 without any issues.

Recently we tried to upgrade to head of Kirkstone to get kernel 5.10, enabling some other features that are needed for our system.
However upon doing this we are no longer able to boot up our device,  we get the following message on boot up:
![20221003_133255](https://user-images.githubusercontent.com/35603661/194234025-08ed72f4-5910-433d-8977-4c3fa2d6e277.jpg)

There does seem to have been a change in how the boot system is set up, but we are unable to figure out how to get around this. Has anyone had a similar issue or know what we can do?

**Comment by madisongh** [comment]
Besides the entirely new bootloader implementation, there are quite a few other changes you may need to account for.  There are links to the NVIDIA pages for L4T R35.1.0 and JetPack 5.0 on the [wiki page](https://github.com/OE4T/meta-tegra/wiki/JetPack-5.0.2-L4T-R35.1.0-Notes) summary of the update.

The screen shot looks normal enough for the new bootloader.  If Linux isn't coming up after that, I'd suggest hooking up the serial console to monitor the boot progress and see where it might be getting stuck.
